### PR TITLE
Azure Container Apps - Support Guide Links

### PR DESCRIPTION
**Pull request by black-bryan**
Fixes to the Azure Kubernetes Service links and Azure Container Apps links. 

Aligning the Azure Container Apps README.md to point customers to the support guide for standard deployments.

### CODE REVIEW

**Review by accraw — APPROVED** [review]
Looks reasonable to me!

File: azure-container-apps/ReadMe.md [hunk 1]
```diff
@@ -11,146 +11,20 @@ The deployment consists of two [containers](https://learn.microsoft.com/en-us/az
 
 **Table of contents:**
 
-- [Overview](#Overview)
-- [Prerequisites](#Prerequisites)
-- [Getting Started: Generate credentials for automated user provisioning with 1Password](#Generate-credentials-for-automated-user-provisioning-with-1Password)
-- [Azure Container App Deployment steps using the Azure Shell](#azure-container-app-deployment-steps-using-the-azure-shell)
+- [Standard Deployment](#standard-deployment)
 - [Update your SCIM bridge](#update-1password-scim-bridge)
```
Comment (ag-adampike) [inline azure-container-apps/ReadMe.md:15]: Maybe we can put all the below list items in an Appendix and link to that section from the "advanced references" link in `\README.md`?

File: README.md [hunk 2]
```diff
@@ -21,7 +21,8 @@ Before you begin deploying 1Password SCIM Bridge, review the [Preparation Guide]
 
 Containers as a service (CaaS) can simplify your deployment by using the built-in tools of the CaaS for DNS and certificate management. This gives you an easy, low-cost SCIM bridge with minimal infrastructure management requirements.
 
-- [Azure Container Apps](/azure-container-apps)
+- [Azure Container Apps](https://support.1password.com/scim-deploy-azure/)
+  - [Container Apps advanced references](/azure-container-apps)
```
Comment (ag-adampike) [inline README.md:25]: I feel like we should say something besides "advanced references". Maybe "advanced topics"? Perhaps Jacob will have a better idea. 😂

**Review by ag-adampike — APPROVED** [review]
A couple suggestions that we could easily consider during Jacob's refinements if you want to merge as is.

File: azure-container-apps/ReadMe.md (hunk 1 above)
Comment (black-bryan) [reply azure-container-apps/ReadMe.md:15]: I think we should review this during Jacob's review on the best way to present this info.

File: README.md (hunk 2 above)
Comment (black-bryan) [reply README.md:25]: Again, hoping during Jacob's review of the documentation at large he will have a better wording selection here.

**Pull request merged**